build.sh

```diff
@@ -25,11 +25,21 @@ readonly source_root="$(dirname "$0")"
 
 PATH=/usr/libexec:$PATH
 
-if [ -n "${JENKINS_HOME}" ]; then
+if [ -n "${CI}" ]; then
     XCPRETTY_PARAMS=(--no-utf --report junit --output build/reports/junit.xml)
     CODESIGN_PARAMS=(CODE_SIGN_IDENTITY='' CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO)
 fi
 
+if [ $CI ]; then
```

Comment:
```suggestion
if [ -n "$CI" ]; then
```

to avoid weird things if $CI has a space in it.